_posts/blog/2020-06-03-welcome-jane-and-teor.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title: "The Zcash Foundation Welcomes Jane and Teor"
+excerpt: "Our engineering team grows to accelerate our roadmap."
+categories: blog
+tags: [team]
+date: 2020-06-03
+author: acityinohio
+---
+
+I'm thrilled to announce that the Zcash Foundation just hired two talented Rust
+Systems Engineers to help us meet the goals of our ambitious [2020 tech
+roadmap](/blog/eng-roadmap-2020/).
+
+## Meet Jane Lusby
+
+[Jane](https://twitter.com/yaahc_) has worked as a C++ systems software engineer
+for embedded and distributed systems. Her professional experience is primarily
+in networking and storage technologies. Her past projects include work on
+Quadcopters, distributed sensor networks, and clustered hypervisors.
+
+In her free time Jane has been most involved in the Rust open source community.
+She has contributed heavily to the rust error handling ecosystem. In addition
+she is a member of the clippy team, wg-traits, a regular tracing contributor,
+and maintains the
+[`awesome-rust-mentors`](https://rustbeginners.github.io/awesome-rust-mentors/)
+project.
+
+At the Foundation, Jane will be primarily focused on
+[Zebra](https://github.com/ZcashFoundation/zebra), our consensus-compatible
+Rust-rewrite Zcash node software, while she will also bring her extensive
+experience with the Rust community to guide future open source contribution and
+development strategy.
+
+## Meet Teor
+
+Teor worked as a software engineer on privacy-preserving protocols and
```

Comment:
Grammar tweak. I'm ok either way, but I think the flow is slightly better with this change.

```suggestion
Teor has worked as a software engineer on privacy-preserving protocols and
```

Reply:
Do I need to mention languages here?
It was mainly C, but I've also worked in Python and Rust, in roughly that order.

(I don't mind either way, and I'm happy to leave them out.)

Reply:
@teor2345 your preference! If you'd like to add go for it, and I'll commit later tonight/tomorrow morning.

Reply:
I think I'm good, it doesn't flow well in my head. And people can work it out if they want :-)